api4/cloud.go

```diff
@@ -138,7 +138,11 @@ func getCloudProducts(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	products, err := c.App.Cloud().GetCloudProducts(c.AppContext.Session().UserId)
+	query := r.URL.Query()
+	_, includeLegacyProducts := query["include_legacy"]
```

Comment:
Do we need to check `includeLegacyProducts` for empty or the actual `true` string?

`includeLegacyProduct := r.URL.Query().Get("include_legacy") == "true"`

Reply:
Yeah, we should.